### PR TITLE
Update Space Colony, using arrow pointer, adding parameters to run

### LINKED ISSUE #1176: Update "Space Colony" script
The following should be done:
- change `var` to `const` or `let`
- change `function` to the arrow syntax
- add the missing parameters to `wine.run`

https://github.com/PhoenicisOrg/scripts/blob/5fe4e1f8a9e0f1003e1e9c523c4afb61d15042f7/Applications/Games/Space%20Colony/Local/script.js#L15-L24

### PR DESCRIPTION
**Pull request by Zemogiter**
### Description
fix #1176 
### What works
Game Installation, downloading and installing the patch
### What was not tested
Nothing
### Test
- Operating system (and linux kernel version): ubuntu 19.10
- Hardware (GPU/CPU): gtx 1080ti, i7-7700K

### Ready for review
- [x] Script tested as a regular phoenicis user and working (if you have a problem -> create as draft and ask for help).
- [x] `json-align` and `eslint` run according to the [documentation](https://phoenicisorg.github.io/scripts/General/tools/). 
- [x] Codacy and travis checked.

### CODE REVIEW

File: Applications/Games/Space Colony/Local/script.js [hunk 1]
```diff
@@ -12,13 +12,13 @@ new LocalInstallerScript()
...
+    .postInstall((wine) => {
         const patch = new Resource()
             .wizard(this._wizard)
             .url("https://d1ztm8591kdhlc.cloudfront.net/hdpatches/Space_Colony_HD_Update.exe")
             .checksum("c821e5c7035b9b517823466f4cedadd3")
             .algorithm("MD5")
             .name("Space_Colony_HD_Update.exe")
             .get();
-        wine.run(patch);
+        wine.run(patch, null, false, true);
```
Comment (madoar) [inline Applications/Games/Space Colony/Local/script.js]: This is a general comment:

Do we want to force the script developers to provide all these parameters or do we want to allow for a shorthand version `wine.run(patch);`? According to my findings it seems like we still have a lot of scripts that use the shorthand form

File: Applications/Games/Space Colony/Local/script.js [hunk 2]
```diff
@@ -12,13 +12,13 @@ new LocalInstallerScript()
     .wineDistribution("upstream")
     .wineVersion(getLatestDevelopmentVersion)
     .executable("Space Colony.exe")
-    .postInstall(function (wine /*wizard*/) {
+    .postInstall((wine) => {
         const patch = new Resource()
             .wizard(this._wizard)
```
Comment (madoar) [inline Applications/Games/Space Colony/Local/script.js]: `.wizard(wine.wizard())`

Comment (madoar) [reply Applications/Games/Space Colony/Local/script.js]: You are missing the `()` at the end

File: Applications/Games/Space Colony/Local/script.js (hunk 1 above)
Comment (Zemogiter) [reply Applications/Games/Space Colony/Local/script.js]: That depends on what those parameters do and if any of them can crash the exe file in question.

**Pull request merged**